services/storage/src/simcore_service_storage/db_access_layer.py

```diff
@@ -112,31 +114,58 @@ def _aggregate_access_rights(
         return AccessRights.none()
 
 
+def assemble_array_groups(user_group_ids: list[GroupID]) -> str:
+    return (
+        "array[]::text[]"
+        if len(user_group_ids) == 0
+        else f"""array[{', '.join(f"'{group_id}'" for group_id in user_group_ids)}]"""
+    )
+
+
+access_rights_subquery = (
```

Comment:
Q: why is this in global scope? or should it not be a capitalized Final constant?

Reply:
I guess thats on me, if you look a bit bellow it is reused in 2 different places.